cdk_infra/lib/app.ts

```diff
@@ -19,4 +20,7 @@ new MSKClustersStack(app, 'msk-clusters', {
   env: envDefault,
   eksVPC: vpcStack.vpc
 });
+new ConfigMapProvidersStack(app, 'config-map-providers', {
+  suffix: process.env.CONFIG_BUCKET_SUFFIX
```

Comment:
Relying on an environment variable does not allow for deterministic deployments. Is it possible that we can hard the required suffix in the stack? Or is there a behavior here that we desire for non deterministic naming like this?

Reply:
Is the purpose of the suffix to allow deployment of this stack in a dev account? If so can we update the naming or add documentation relating to this purpose?

Reply:
Adventures in going through all the files. I now see the comment in the `stack` file. 

nit: I  still am not a fan of relying on environment variables even for dev and testing purposes. I would prefer if instead of relying on env variables it was hard coded in the app. I would not hold the PR up for it.

Reply:
It is in the comments:
>   // Suffix used to test the stack and avoid collisions with s3 buckets names, that must be globally unique.

Reply:
We cannot get rid of the environment variable because of local tests. I hid the environment variable inside the stack instead of leaking into the app.

Reply:
also, I'm happy to hear alternatives for the case of local tests in a dev account.

Reply:
In scenarios like this, one thing I have done  is to not rely on a bucket name without a UUID. Even in the default, non dev account, scenario. Have we considered attaching the account number as a UUID to the bucket?

Reply:
I guess it makes sense to use the account id as a dedup mechanism since it is is only going to be used for the tests.

the downside is that we will need to fetch the account id in the tests and it might create confusion in case the account used for holding the configurations is not the same as the accounts used to run the tests. I guess this is ok.

Reply:
You can derive the account ID from inside the stack. Could [this](https://github.com/aws/aws-cdk/issues/1754#issuecomment-513542145) possibly be a good solution?

Reply:
Also, I'm not sure how this "could" break unit tests or how unit tests handle account env values. Can you make an account id in the unit tests?

Reply:
The tests that I meant are the integration tests.